python/arcticdb/adapters/lmdb_library_adapter.py

```diff
@@ -162,14 +161,12 @@ def create_library(self, name, library_options: LibraryOptions):
 
         return lib
 
-    def cleanup_library(self, library_name: str, library_config: LibraryConfig):
-        for k, v in library_config.storage_by_id.items():
-            lmdb_config = LmdbConfig()
-            v.config.Unpack(lmdb_config)
-            shutil.rmtree(os.path.join(lmdb_config.path, library_name), onerror=_rmtree_errorhandler)
+    def cleanup_library(self, library_name: str):
```

Comment:
Side note: this PR makes a breaking change to the public API (like here with removing a parameter to this function), so according to SemVer, this should _stricto sensus_ be part of 5.0.

I am not sure that we want to follow semantic versioning at all cost. Henry Schreiner mentions [realistic semantic versioning](https://iscinumpy.dev/post/bound-version-constraints/#semver). Might this be relevant for ArcticDB?

Reply:
I think we're regarding Arctic, Library and QueryBuilder as our public API. Those are the only documented parts of the API. https://docs.arcticdb.io/api/